hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGrantKycToAccountHandlerTest.java

```diff
@@ -221,16 +234,20 @@ void nullAccountIdThrowsException() {
         }
 
         @Test
-        @DisplayName("When getForModify returns empty, should not put or commit")
+        @DisplayName("When TokenRelStore.get() returns empty, should not put or commit")
         void emptyGetForModifyShouldNotPersist() {
```

Comment:
Function name should match display name modification here as well
```suggestion
        @DisplayName("When TokenRelStore.get() returns empty, should not put or commit")
        void emptyGetShouldNotPersist() {
```